README.md

```diff
@@ -97,7 +97,7 @@ Protocol Buffer description. This is mainly for use in testing and debugging.
 that reports some basic information about an API. The "-" causes the plugin to 
 write its output to stdout.
 
-        go install github.com/googleapis/gnostic/plugins/gnostic-go-generator
+        go install github.com/googleapis/gnostic-go-generator
```

Comment:
maybe this and the following line should go too

Reply:
Might be good to have an example, that's what this section of the readme is about. Should I switch it to one that exists in the repo? e.g. swift?

Happy to remove it altogether though.

Reply:
Removed it an rephrased the section.

Reply:
LGTM! IMO the Swift repo also needs to go into a separate repo.